appendix-basic65.tex

```diff
@@ -4375,20 +4455,19 @@ \subsection{FOR}
 % **********
 
 \newpage
-\label{foregroundcommand}
 \subsection{FOREGROUND}
```

Comment:
Just curious on the backstory of the removal of the label?

Reply:
It is unused.

For BASIC commands with labels, I'm standardizing on having the label match the index term, instead of inventing a new syntax every time someone needs a label. I went to change this one and noticed nothing refers to it, so I removed it.